README.md

```diff
@@ -1,56 +1,20 @@
+_[Ably](https://ably.com) is the platform that powers synchronized digital experiences in realtime. Whether attending an event in a virtual venue, receiving realtime financial information, or monitoring live car performance data – consumers simply expect realtime digital experiences as standard. Ably provides a suite of APIs to build, extend, and deliver powerful digital experiences in realtime for more than 250 million devices across 80 countries each month. Organizations like Bloomberg, HubSpot, Verizon, and Hopin depend on Ably’s platform to offload the growing complexity of business-critical realtime data synchronization at global scale. For more information, see the [Ably documentation](https://ably.com/documentation)._
 
 ## Overview
 
 A [Flutter](https://flutter.dev/) plugin for [Ably](https://www.ably.com),
 built on top of Ably's [iOS](https://github.com/ably/ably-cocoa) and [Android](https://github.com/ably/ably-java) SDKs.
 
-## Resources
-
-- [Quickstart Guide](https://www.ably.com/docs/quick-start-guide?lang=flutter)
-- [Introducing the Ably Flutter plugin](https://www.ably.com/blog/ably-flutter-plugin) by [Srushtika](https://github.com/Srushtika) (Developer Advocate)
-- [Building a Realtime Cryptocurrency App with Flutter](https://www.ably.com/tutorials/realtime-cryptocurrency-app-flutter) by [pr-Mais](https://github.com/pr-Mais) and [escamoteur](https://github.com/escamoteur)
-- [Building realtime apps with Flutter and WebSockets: client-side considerations](https://www.ably.com/topic/websockets-flutter)
-
-## Supported Platforms
-
-### iOS
-
-iOS 10 or newer.
-
-### Android
-
-API Level 19 (Android 4.4, KitKat) or newer.
-
-_This project uses Java 8 language features, utilising [Desugaring](https://developer.android.com/studio/write/java8-support#library-desugaring)
-to support lower versions of the Android runtime (i.e. API Levels prior to 24)_
-
-If your project needs support for SDK Version lower than 24, Android Gradle Plugin 4.0.0+ must be used.
-You might also need to upgrade [gradle distribution](https://developer.android.com/studio/releases/gradle-plugin#updating-plugin) accordingly.
-
-## Updating to a newer version
-
-When increasing the version of `ably_flutter` in your `pubspec.yaml`, there may be breaking changes. To migrate code across these breaking changes, follow the [updating / migration guide](UPDATING.md).
-
-## Known Limitations
-
-Features that we do not currently support, but we do plan to add in the future:
-
-- Ably token generation ([#105](https://github.com/ably/ably-flutter/issues/105))
```

Comment:
I noticed that you've removed, rather than moved, this list. A couple of these issues are closed, but the rest remain open, so I think they should remain highlighted in the readme in their new location.

Reply:
I wanted to move it, but it seems I missed it somehow, good catch 👍  I've restored it in https://github.com/ably/ably-flutter/pull/413/commits/6f8b92a7b21c656bdf408a2414fe80b7f2d466c3